config.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const path = require('path');
+const NODE_ENV = process.env.NODE_ENV;
+
+const MgProcesses = require('./lib/mg_process.js');
```

Comment:
Is it conventional to capitalise variables containing imported modules?

Reply:
["A JavaScript class is declared with a PascalCase in contrast to other JavaScript data structures"](https://www.robinwieruch.de/javascript-naming-conventions).

I've made a mistake here though. `MgProcess` isn't a `class`, I'll change.